### PR TITLE
Auto building Windows Exe and WebGL for itch.io

### PR DESCRIPTION
**Pull request by MaXeraph**
- Builds only on [production](https://github.com/MaXeraph/Adrenarun/tree/production)
- Adds linting to new files in the PR
- Windows EXE and html5 WebGL
- Lighting is not "real" but it has a minimum brightness level. Basically the same and I don't anticipate any issues with our goals

### CODE REVIEW

File: Assets/Scenes/LevelPrototypeScene.unity [hunk 1]
```diff
@@ -20,11 +20,11 @@ RenderSettings:
   m_FogDensity: 0.01
   m_LinearFogStart: 0
   m_LinearFogEnd: 300
-  m_AmbientSkyColor: {r: 0.212, g: 0.227, b: 0.259, a: 1}
+  m_AmbientSkyColor: {r: 0.5874866, g: 0.64951944, b: 0.7735849, a: 1}
```
Comment (zhuandy42) [inline Assets/Scenes/LevelPrototypeScene.unity:23]: what happened here

Comment (MaXeraph) [reply Assets/Scenes/LevelPrototypeScene.unity:23]: I had to hardcode the lighting color in order to fix the dark lighting issue.

Comment (MaXeraph) [reply Assets/Scenes/LevelPrototypeScene.unity:23]: Again dont expect it to be an issue later on

**Comment by MaXeraph** [comment]
Added new Linting check for newly edited files

**Pull request merged**